### DISCUSSION TITLE
Skript Coordinates

### DISCUSSION THREAD

**Discussion by MrEquinOfficial** [SkriptLang/Skript#6441] [General]
Hello. I was writing a plugin for the RPG server and I have a problem. I only want players in a specific region to break oak_log. But even if the command works without any problems, it can break everywhere when I try it in the game. It's not just between these 4 coordinates. What is the problem and how can I fix it?
`
on break:
    set {player_x} to player's x position
    set {player_z} to player's z position
    if {player_x} is between 168 and 187:
        if {player_z} is between -658 and -682:
            if event-block is oak log:
                #Sa
            else:
                cancel event
                send "&4&o Sən burada blok qıra bilmərsən!" to player
`

**Comment by AyhamAl-Ali** [comment]
For Skript coding help please use [skUnity discord help channels](https://discord.gg/skript). If you are having a bug/feature request report it here.
